worker/src/main/resources/application.properties

```diff
@@ -19,6 +19,9 @@ cancellation.check.frequency=10
 report.progress.db.frequency=100
 report.progress.log.frequency=10000
 
+## fail the job if >= 10% of the records fail
+failure.threshold=10
```

Comment:
Let's start with a conservative default of 1%. Still a large number for large contracts.